tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/ImplementationTimescales/Suppliers Edit Solution Implementation Timescales Section.3670.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Suppliers Edit Solution Implementation Timescales Section
+    As a Supplier
+    I want to Edit the Solution Implementation Timescales Section
+    So that I can make sure the information is correct
+
+    Background:
+        Given Suppliers exist
+            | Id    | SupplierName |
+            | Sup 1 | Supplier 1   |
+        And Solutions exist
+            | SolutionID | SolutionName   | SupplierStatusId | SupplierId |
+            | Sln1       | MedicOnline    | 1                | Sup 1      |
+            | Sln2       | TakeTheRedPill | 1                | Sup 1      |
+            | Sln3       | PracticeMgr    | 1                | Sup 1      |
+        And SolutionDetail exist
+            | Solution | ImplementationTimescales                              |
+            | Sln1     | An original implementation timescales description      |
+            | Sln2     | Another original implementation timescales description |
+
+    @3670
+    Scenario: 1. Solution implementation timescales section data is updated
+        When a PUT request is made to update the implementation-timescales section for solution Sln1
+            | ImplementationTimescales                    |
+            | A new implementation timescales description |
+        Then a successful response is returned
+        And SolutionDetail exist
```

Comment:
This needs to be completed. Go to where solution detail is and include this

Reply:
I don't know what you mean by this

Reply:
This was sorted with the bug fix. Editing the actual step of solutionDetail